ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -157,41 +160,108 @@ namespace ngraph
             return std::make_shared<ngraph::Function>(result_nodes, parameter_nodes);
         }
 
-        InputModel::Ptr FrontEndPDPD::load_from_file(const std::string& path) const
-        {
-            return load_from_files({path});
-        }
-
-        InputModel::Ptr FrontEndPDPD::load_from_files(const std::vector<std::string>& paths) const
+        bool FrontEndPDPD::supported_by_variants(
+            const std::vector<std::shared_ptr<Variant>>& variants) const
         {
-            if (paths.size() == 1)
+            // Validating first path, it must contain a model
+            PDPD_ASSERT(variants.size() > 0, "Variants should contain at least one path/stream");
+            if (is_type<VariantWrapper<std::string>>(variants[0]))
             {
-                // The case when folder with __model__ and weight files is provided or .pdmodel file
-                return std::make_shared<InputModelPDPD>(paths[0]);
+                std::string suffix = ".pdmodel";
```

Comment:
previously there was support of 2 parameters `if (paths.size() == 2)`, where it was possible to pass '.pdmodel' and '.pdiparams'. Is it supported now?